### ISSUE TITLE
Subtitles not being autopopulated when rerecording part of a slice

### ISSUE THREAD

**Issue by artjomsR** [asbplayer/asbplayer#587]
**What is the issue and how what do you do to make it happen**

I originally discovered this by working with subtitles that had too much slack after the actual audio and wanted to trim them.

- Steps to reproduce:
  1) Press `Mine current subtitle and open Anki dialogue`shortcut  
  2) Notice the Subtitle Track field being populated  
  3) On the timeline below, make the selection smaller, such that it becomes part of the current subtitle slice
  4) Rerecord audio
  Outcome: Subtitle Track field has no text, despite containing part of the original audio
  
   I think rather than having no text for Subtitle Track, it makes more sense to have the original text before the trimming: it is easier to edit this text if desired, rather than typing from scratch

https://github.com/user-attachments/assets/f1b1a98c-3de8-45fe-aa5d-bb1f64f21669

**What browser?**
Version 128.0.6613.189 (Official Build) (64-bit)

**Is it happening with local video or streaming video?**
Streaming

**If the issue is happening with streaming video, what version of the extension are you using?**
commit 6a41684c2

**If the issue is happening with streaming video, on what website(s) is the issue happening?**
Youtube

**Comment by killergerbah** [comment]
@artjomsR I don't think this has to do with re-recording, it's just the current behavior of how adjusting the slider automatically updates the subtitle text fields. Maybe your problem could be solved by allowing the auto-update behavior to be turned off under some conditions.
